### PR TITLE
Stack analyses 1st attempt

### PR DESCRIPTION
**Pull request by tisnik**
Quick check that stack-analysis endpoint accepts project.json files

### CODE REVIEW

File: integration-tests/features/stack_analyses.feature [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+  Scenario: Check that the API entry point
+    Given System is running
+    When I access /api/stack-analyses
+    Then I should get 404 status code
+
+  Scenario: Check that the API entry point
+    Given System is running
+    When I wait 10 seconds
+    Then I wait 10 seconds
```
Comment (jpopelka) [inline integration-tests/features/stack_analyses.feature]: I'm just curious, what's the purpose of this `When I wait ...` - `Then I wait ...` ?

Comment (tisnik) [inline integration-tests/features/stack_analyses.feature]: Well the problem is that component-analyses and stack-analyses endpoints are ready to accept API calls after some time, even when the system seems to be started properly. I talked about this issue with @fridex and ATM it seems there's no proper way how to ensure that the queue/workers will accept commands via API calls. So this is quick & dirty (and ugly as well) 1st attempt how to let the system to take a breath ;) I'm going to propose slight API design change to do such tests properly.

Comment (jpopelka) [inline integration-tests/features/stack_analyses.feature]: Ok, that's what I was thinking.
But as I understand it, both `When I wait 10 seconds` and `Then I wait 10 seconds` do `time.sleep(10)`, so why we have both these lines here ? For example only `Then I wait 20 seconds` would do the same, wouldn't it ?

Comment (tisnik) [inline integration-tests/features/stack_analyses.feature]: I'm pretty sure you are right, let me check

**Comment by tisnik** [comment]
@jpopelka better? :) Btw the stack_analyses tests are not enabled by default ATM, ie. the feature file is not mentioned in feature_list.txt (on purpose)

**Pull request merged**